lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/dcmaw-driving-licence.yaml

```diff
@@ -0,0 +1,112 @@
+name: DCMAW and Driving Licence
```

Comment:
I'm not sure I like this name (sorry!) - it supports a range of docs, it just so happens that DL will perform an extra check.

Perhaps something like `APP_DOC_CHECK`, and the description can add a comment that it 'may' perform authoritative source checks on certain document types.